### DISCUSSION TITLE
Building with `gptfdisk` fails in installation step with `find: ‘./packages/target-mips_gcc-5.5.0_uClibc-1.0.14-nptl_kernel-3.10/gptfdisk-d7f3d306b083123bcc6f5941efade586/’: No such file or directory`.

### DISCUSSION THREAD

**Discussion by dreirund** [Freetz-NG/freetz-ng#978] [Allgemein]
When I want to build "Freetz-NG" with `gptfdisk`, build fails while installing packages with `find: ‘./packages/target-mips_gcc-5.5.0_uClibc-1.0.14-nptl_kernel-3.10/gptfdisk-d7f3d306b083123bcc6f5941efade586/’: No such file or directory`:

`make`:  
```
[...]
installing packages
  [...]
  gnu-make-4.2.1 ....................   72.33 Kb 
  gptfdisk-d7f3d306b083123bcc6f59 ...find: ‘./packages/target-mips_gcc-5.5.0_uClibc-1.0.14-nptl_kernel-3.10/gptfdisk-d7f3d306b083123bcc6f5941efade586/’: No such file or directory
/home/<censored>/download/Router/AVM/FritzBox_7362SL/firmware/freetz-ng/freetz-ng/tools/tar-gnu: packages/target-mips_gcc-5.5.0_uClibc-1.0.14-nptl_kernel-3.10/gptfdisk-d7f3d306b083123bcc6f5941efade586/root: Cannot open: No such file or directory
/home/<censored>/download/Router/AVM/FritzBox_7362SL/firmware/freetz-ng/freetz-ng/tools/tar-gnu: Error is not recoverable: exiting now
tar: short read
ERROR: tar failed
FINISHED
```  
*(I have censored my local user name.)*

`grep GPTFDISK .config`:  
```
FREETZ_PACKAGE_GPTFDISK=y
```

Disabling `FREETZ_PACKAGE_GPTFDISK` makes build of "Freetz-NG" continuing.

I am aware that `gptfdisk` is in the "Unstable"-menu and thus not expected to work, I want to report it anyway in case you might be interested in feedback of actual errors.

* Freetz-NG version: Freetz-NG 23565M-f2a453e28 master 2024-04-18,
* Attached: [`.config`](https://github.com/Freetz-NG/freetz-ng/files/15038585/default.config.txt).

Regards!

**Comment by fda77** [comment]
what did you do to investigate?

**Comment by dreirund** [comment]
> what did you do to investigate?

What do you mean?

Nothing.

I just run "Freetz-NG"'s build system and reported it here in case
anyone is interested.

**Comment by fda77** [comment]
In case you can read, please read again what you read when you tried to open an issue

https://github.com/Freetz-NG/freetz-ng/blob/master/.github/ISSUE_TEMPLATE/1-bug_report.md

**Reply by fda77** [reply]
For easier reading, full quote of the file: 

>> Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem! Issues are no longer accepted, please create a pull-request to fix your problem!

**Reply by dreirund** [reply]
I know, but it is beyond my capabilities to open a pull request.

And I do not see it as "I want my problem be fixed", but as "I encountered an issue and I want let you know about it, maybe you find it valuable".  
If you do not want to be informed about issues -- well, OK then.

I happily can live without those packages on my router.

Regards!
